Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/recyclerview/ThrottledClickListener.kt

```diff
@@ -0,0 +1,22 @@
+package de.rki.coronawarnapp.util.recyclerview
+
+import android.view.View
+
+/*  Prevents a recyclerview item to be double selected when double clicking on it  */
+class ThrottledClickListener(
+    private val interval: Long,
```

Comment:
you can set a default of 300 so you don't have to set it when calling the function above.